### PR TITLE
ACS-6393: [HxI PH] Feedback protection in Live Ingester

### PR DESCRIPTION
**Pull request by krdabrowski**
https://hyland.atlassian.net/browse/ACS-6393

### CODE REVIEW

File: live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/repository/util/EventUtils.java [hunk 1]
```diff
@@ -78,4 +85,27 @@ public static EventType getEventType(RepoEvent<DataAttributes<NodeResource>> eve
...
+        Set<String> aspects = Optional.ofNullable(event.getData().getResource().getAspectNames()).orElse(Collections.emptySet());
+        if (aspects.contains(PREDICTION_APPLIED_ASPECT))
+        {
+            String actualPredictionTime = (String) event.getData().getResource().getProperties().get(PREDICTION_TIME_PROPERTY);
+            String beforePredictionTime = (String) Optional.ofNullable(event.getData().getResourceBefore())
+                    .map(NodeResource::getProperties)
+                    .map(properties -> properties.get(PREDICTION_TIME_PROPERTY))
+                    .orElse(null);
+
+            return actualPredictionTime != null && actualPredictionTime.equals(beforePredictionTime)
```
Comment (tpage-alfresco) [inline live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/repository/util/EventUtils.java:105]: I hadn't really thought about the situation where we remove the prediction time from the node. This is fine for now, but I think we'll revisit this when we work on https://hyland.atlassian.net/browse/ACS-7708

**Review by tpage-alfresco — APPROVED** [review]
Ideally we'd also have an e2e test that changing `hxi:latestPredictionDateTime` suppresses the event, but on the whole this looks really good :+1:

**Review by mpichura — APPROVED** [review]
LGTM

File: live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/repository/util/EventUtils.java (hunk 1 above)
Comment (krdabrowski) [reply live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/repository/util/EventUtils.java]: I've updated the logic

**Comment by krdabrowski** [comment]
I will add E2E test in a separate PR

**Pull request merged**